packages/ciscospark/package.json

```diff
@@ -12,6 +12,8 @@
     "@ciscospark/http-core": "^0.7.18",
     "@ciscospark/plugin-logger": "^0.7.18",
     "@ciscospark/plugin-phone": "^0.7.18",
+    "@ciscospark/plugin-board": "^0.7.18",
+    "@ciscospark/plugin-conversation": "^0.7.18",
```

Comment:
I don't think we want to distribute this directly with `ciscospark`. It'll pull in plugin-encryption and plugin-credentials. Encryption is big and Credentials is very new (with some potential lockout-inducing error cases that need to be proven before shipping). As I understand it, this is a temporary measure until the public API has board support. 

We can alternatively implement this as e.g. `plugin-whiteboard` and tell people who to need it to get started with something like
```javascript
import '@ciscospark/plugin-whiteboard';
import ciscospark from 'ciscospark';
```